mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_cluster_sweeper.go

```diff
@@ -0,0 +1,130 @@
+package vmwareengine
+
+import (
+	"context"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/envvar"
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/sweeper"
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google-beta/google-beta/transport"
+)
+
+func init() {
+	sweeper.AddTestSweepers("VmwareengineCluster", testSweepVmwareengineCluster)
+}
+
+// At the time of writing, the CI only passes us-central1 as the region
+func testSweepVmwareengineCluster(region string) error {
+	resourceName := "VmwareengineCluster"
+	log.Printf("[INFO][SWEEPER_LOG] Starting sweeper for %s", resourceName)
+
+	config, err := sweeper.SharedConfigForRegion(region)
+	if err != nil {
+		log.Printf("[INFO][SWEEPER_LOG] error getting shared config for region: %s", err)
+		return err
+	}
+
+	err = config.LoadAndValidate(context.Background())
+	if err != nil {
+		log.Printf("[INFO][SWEEPER_LOG] error loading: %s", err)
+		return err
+	}
+
+	t := &testing.T{}
+	billingId := envvar.GetTestBillingAccountFromEnv(t)
+
+	// List of location values includes:
+	//   * global location
+	//   * regions used for this resource type's acc tests in the past
+	//   * the 'region' passed to the sweeper
+	locations := []string{"global", "us-central1", "southamerica-west1", "me-west1", region}
```

Comment:
```suggestion
	locations := []string{region, "global", "southamerica-west1", "me-west1"}
```

Use user-provided region first, remove us-central1 as it's assumed that's the value of region most times